app/src/test/java/com/eveningoutpost/dexdrip/RobolectricTestWithConfigAndPowerMockLogRedirect.java

```diff
@@ -1,112 +0,0 @@
-package com.eveningoutpost.dexdrip;
```

Comment:
Can you undelete this file. It is a useful example I think of how to get powermock and roboelectric actually working together. We don't need to use it but I don't think it should be deleted.